package.json

```diff
@@ -6,17 +6,17 @@
     "url": "https://github.com/seek-oss/logger-js/issues"
   },
   "dependencies": {
-    "@types/pino": "^6.3.2",
-    "dtrim": "^1.9.0",
-    "pino": "^6.6.0",
-    "pino-std-serializers": "^2.5.0"
+    "@types/pino": "6.3.2",
+    "dtrim": "1.9.0",
+    "pino": "6.7.0",
+    "pino-std-serializers": "2.5.0"
```

Comment:
Would be nice to keep the `dependencies` as carets so we don't have to republish for every new minor/patch

Reply:
true